### DISCUSSION TITLE
Amazon Stock unavailable

### DISCUSSION THREAD

**Discussion by Lieuwe** [revk/xESP32-Faikout#154] [General]
When will the amazon stock be available again?
I would like to order one.

**Comment by revk** [comment]
They have been shipped, received by Amazon on 22nd, and are being processed by Amazon now, it should be any day.

**Comment by 44616E** [comment]
G'day @revk , looking to order 6 more and see Amazon is out of stock again. Do you know when they may be in stock again?

I will also be adding some new model numbers to the list, doesn't seem there are any Australian units on there yet. Working fine on our 9.5kW Premium XL even with the inbuilt Wifi still turned on. Thanks for the great product.

**Comment by revk** [comment]
I am expecting 600 boards this week, so should have plenty of stock. But it will take a few days more for Amazon to get them on the site. Sorry for the delay.
